websites/C/ChompuBot/presence.ts

```diff
@@ -0,0 +1,141 @@
+const presence = new Presence({
+		clientId: "1219713910165209169",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://chompubot.work/logo.jpg",
+}
+
+const enum Pages {
+	Home = "/",
+	Commands = "/commands",
+	Dashboard = "/dashboard",
+	FAQ = "/faq",
+	Contact = "/contact",
+}
+
+let presenceData: PresenceData;
+
+presence.on("UpdateData", async () => {
+	const base = document.location.pathname;
+
+	if (/\/dashboard\/guild\/(.*[0-9_].*)\/music-room/gi.test(base)) {
+		let username,
+			title,
+			author,
+			playing,
+			timeStartPlayer,
+			timeEndPlayer,
+			startPlayer,
+			durationPlayer,
+			startTimestamp,
+			endTimestamp;
+
+		const status = document.querySelector<HTMLElement>(
+			"div.hidden.-player-status"
+		).textContent;
+		while (status === "loading")
+			await new Promise(resolve => setTimeout(resolve, 250));
```

Comment:
updatedata gets called like every 100ms so maybe just return if status is loading...

Reply:
i can settimout 1000 ?

Reply:
remove it completely